exec/java-exec/src/test/java/org/apache/drill/TestSchemaWithTableFunction.java

```diff
@@ -17,6 +17,14 @@
  */
 package org.apache.drill;
 
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
```

Comment:
I am not sure why your IDE makes static imports on the top of the list, I think its more common when they are in the end. I think all Drill project follows this convention...

Reply:
Sorry, I've manually put the imports back and have updated the Eclipse rules to put static imports after the others.

As it turns out, the Eclipse style rules on the Drill web site are not valid for the latest Eclipse: "Import failed. This is not a faled profile: Expected `CleanUpProfile` but encountered `CodeFormatterProfile`. So, I've been using the Eclipse default profile with a few adjustments per the [web site](http://drill.apache.org/docs/apache-drill-contribution-guidelines/). The web site does not mention a preferred import order.

I have tried disabling the auto-update of imports so that files are left unchanged. However, this tends to leave unused imports which then cause the build to fail and take a long time to fix manually.

Best solution: update the Eclipse settings file both to make it valid and to include any missing style rules. Then, I'll update my IDE to enforce the project's preferred standards.